src/Uno.Sdk/targets/Uno.ProjectCapabilities.targets

```diff
@@ -1,8 +1,12 @@
 <Project>
-	<!-- Sync with https://github.com/dotnet/maui/blob/ffab30545ac146710a9ee61138be33e52ca4b326/src/Templates/src/templates/maui-mobile/Directory.Build.targets -->
 	<PropertyGroup Condition="!$(IsWinAppSdk)">
+		<!-- Sync with https://github.com/dotnet/maui/blob/ffab30545ac146710a9ee61138be33e52ca4b326/src/Templates/src/templates/maui-mobile/Directory.Build.targets -->
 		<!-- Required - Enable Launch Profiles for .NET 6 iOS/Android -->
 		<_KeepLaunchProfiles>true</_KeepLaunchProfiles>
+
+		<!-- Forces the VS Intellisense to detetect uno projects as WinUI XAML -->
+		<!-- https://github.com/unoplatform/uno/issues/15517 -->
+    	<DefaultXamlRuntime>WinUI</DefaultXamlRuntime>
```

Comment:
Can we have this for SamplesApp as well? I hope it will fix the too bad IntelliSense there?

Reply:
We could! Will add it there.